content/documents/search-index-tools.mdx

```diff
@@ -0,0 +1,73 @@
+---
+type:           article
+title:          Introducing Search Index Tools for Portal
+description:    A new set of tools for troubleshooting search index issues
+author:         edustin 
+publishDate:    2023-01-03
+updatedDate:    2023-01-03
+tags:           ["Search", "Portal"]
+---
+
+We're excited to unveil a long-requested enhancement for OrderCloud's premium search feature: a self-service tool for viewing and rebuilding your search indexes. By the end of this article, you should be ready to investigate and resolve basic search errors in your marketplace without the need for OrderCloud support.
+
```

Comment:
comma after marketplace and then "eliminating the need to seek additional support from Sitecore in most cases."